dev/app.py

```diff
@@ -1,8 +1,13 @@
 import lvgl as lv
 from display import Display
 from widgets import *
-
+from icons import *
```

Comment:
Would it be better to `import widgets as w` and `import icons as i` or similar?

That way it is clearer where the respective libraries are used.